adapter/src/main/java/com/twilio/kudu/sql/rules/KuduAggregationLimitRule.java

```diff
@@ -59,9 +59,9 @@
  */
 public class KuduAggregationLimitRule extends RelOptRule {
 
-  private static final RelOptRuleOperand OPERAND = operand(EnumerableLimit.class,
-      some(operand(Sort.class, some(operand(Aggregate.class, some(operand(KuduToEnumerableRel.class,
-          some(operand(Project.class, some(operand(Filter.class, some(operand(KuduQuery.class, none())))))))))))));
+  private static final RelOptRuleOperand OPERAND = operand(Sort.class,
+      some(operand(Aggregate.class, some(operand(KuduToEnumerableRel.class,
+          some(operand(Project.class, some(operand(Filter.class, some(operand(KuduQuery.class, none())))))))))));
```

Comment:
do you need `KuduToEnumerableRel` here? could this be removed

```java
 private static final RelOptRuleOperand OPERAND = operand(Sort.class,
      some(operand(Aggregate.class, some(operand(Project.class, some(operand(Filter.class, some(operand(KuduQuery.class, none())))))))));
```

And should it be `LogicalX` instead? that would match less things and make the rule fire less often

Reply:
I tried making this change to both rules but when I do the plan with the `KuduSortRel` is not chosen. 
Instead of 
```
EnumerableCalc(expr#0..2=[{inputs}], FIELD1=[$t1], FIELD2=[$t0], EXPR$2=[$t2])
  EnumerableAggregate(group=[{0, 1}], EXPR$2=[$SUM0($2)])
    KuduToEnumerableRel
      KuduSortRel(sort0=[$1], sort1=[$2], dir0=[DESC], dir1=[DESC], offset=[1], fetch=[2], groupBySorted=[true])
        KuduProjectRel(FIELD2=[$2], FIELD1=[$1], FIELD4=[$4])
          KuduFilterRel(ScanToken 1=[ACCOUNT_SID EQUAL ACCOUNT1])
            KuduQuery(table=[[kudu, SortedAggregationIT]])
```
I get
```
EnumerableCalc(expr#0..2=[{inputs}], FIELD1=[$t1], FIELD2=[$t0], EXPR$2=[$t2])
  EnumerableLimitSort(sort0=[$1], sort1=[$0], dir0=[DESC], dir1=[DESC], offset=[1], fetch=[2])
    EnumerableAggregate(group=[{0, 1}], EXPR$2=[$SUM0($2)])
      KuduToEnumerableRel
        KuduProjectRel(FIELD2=[$2], FIELD1=[$1], FIELD4=[$4])
          KuduFilterRel(ScanToken 1=[ACCOUNT_SID EQUAL ACCOUNT1])
            KuduQuery(table=[[kudu, SortedAggregationIT]])
```